app/liquid_tags/jsitor_tag.rb

```diff
@@ -1,6 +1,7 @@
 class JsitorTag < LiquidTagBase
   PARTIAL = "liquids/jsitor".freeze
   URL_REGEXP = /\A(https|http):\/\/(jsitor)\.(com)\/(embed)\/([a-zA-Z0-9]+)([?a-zA-Z&]*)\Z/.freeze
+  ID_REGEXP = /\A([?a-zA-Z0-9&])*\Z/.freeze
```

Comment:
I think this regexp is unusually complicated. 

It seems to me what you're generally trying to match is every ASCII char and every digit in addition to the `&` and `?` chars, with no specific length of the whole string. You also don't need grouping.

I think it can be simplified to `\A[\w&?]+\Z` as any word character, plus the `&`, `?`, repeated once or more.

I tested them with Rubular

What do you think?

Reply:
ohh thats cool. yea for sure I didnt know that. thank you. also if I dont group it for some reason it wont validate I tried it on ruby console this part took me long time to figure out because of not grouping it but let me try again.